lib/drivers/insuletDriver.js

```diff
@@ -896,6 +896,13 @@ module.exports = function (config) {
             .with_status(postsuspend)
             .done();
           break;
+        // for alarm codes not documented in the spec
+        default:
+          if (postalarm) {
+            postalarm = postalarm.with_alarmType('other')
```

Comment:
Is it okay for an 'other' alarm type to NOT have a payload with alarmText, explanation, and stopsDelivery (as seen above on line 825)?  If so, LGTM!  :shipit: